components/brave_new_tab_ui/components/default/page/index.ts

```diff
@@ -25,13 +25,13 @@ interface PageProps {
 }
 
 function getItemRowCount (p: PageProps): number {
-  let right = (p.showClock ? 1 : 0) + (p.showRewards ? 2 : 0)
+  let right = (p.showClock ? 1 : 0) + (p.showRewards ? 1 : 0)
   let left = (p.showStats ? 1 : 0) + (p.showTopSites ? 1 : 0)
   // Has space for branded logo to sit next to something on right?
-  if (p.showBrandedWallpaper && left >= right) {
+  if (0 && left >= right) {
```

Comment:
Is this a typo? This statement will never evaluate to true

Reply:
I actually don't need the if statement at all, but I'm not sure if it's okay for me to delete it so I left it like that. Should I delete it?